.gitattributes

```diff
@@ -20,4 +20,7 @@
 *.ico binary
 *.snk binary
 *.pdb binary
-*.svg binary
\ No newline at end of file
+*.svg binary
+
+# Don't check for whitespace errors in the doc pages
+*.md -whitespace
```

Comment:
Markdown uses two blanks at the end of the line to force a linebreak. This option turns off checking for whitespace errors so that a local git client won't complain when you try to commit a file with such lines.
I don't know what GitHub does, I hope it honors those settings (as it should) - if not then it's no worse than before on GitHub, but at least it is an improvement for local git clients.